requirements.txt

```diff
@@ -0,0 +1,20 @@
+appdirs==1.4.0
+click==6.7
+colorama==0.3.7
+coverage==4.3.4
+docopt==0.6.2
+ez-setup==0.9
+feedparser==5.2.1
+nose==1.3.7
+numpydoc==0.6.0
+packaging==16.8
+ply==3.9
+prompt-toolkit==1.0.9
+Pygments==2.1.3
+pyparsing==2.1.10
+pytz==2016.10
+requests==2.12.4
```

Comment:
confirm the modules isted in `requirements.txt` like I haven't seen any implemetation of `requests` `prompt-toolkit`